worlds/kh2/__init__.py

```diff
@@ -128,9 +128,11 @@ def generate_early(self) -> None:
             luckyemblemamount = self.multiworld.LuckyEmblemsAmount[self.player].value
             luckyemblemrequired = self.multiworld.LuckyEmblemsRequired[self.player].value
             if luckyemblemamount < luckyemblemrequired:
+                print(f"Lucky Emblem Amount {self.multiworld.LuckyEmblemsAmount[self.player].value} is less than required "
```

Comment:
leftover debug print? If not, it should use logging

Reply:
changed it to logging.info